Formula/ahoy.rb

```diff
@@ -0,0 +1,35 @@
+class Ahoy < Formula
+  desc "Creates self documenting cli programs from commands in yaml files"
+  homepage "http://www.ahoycli.com/"
+  url "https://github.com/ahoy-cli/ahoy/archive/2.0.0.tar.gz"
+  sha256 "cc3e426083bf7b7309e484fa69ed53b33c9b00adf9be879cbe74c19bdaef027c"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/ahoy-cli/ahoy"
+    bin_path.install Dir["*"]
+    cd bin_path do
+      system "go", "build", "-o", bin/"ahoy", "-ldflags", "-X main.version=#{version}", "."
+    end
+  end
+
+  def caveats; <<~EOPLIST
+    ===== UPGRADING FROM 1.x TO 2.x =====
+
+    If you are upgrading from ahoy 1.x, note that you'll
+    need to upgrade your ahoyapi settings in your .ahoy.yml
+    files to 'v2' instead of 'v1'.
+
+    See other changes at:
+
+    https://github.com/ahoy-cli/ahoy
+
+  EOPLIST
+  end
+
+  test do
+    system "#{bin}/ahoy", "--version"
```

Comment:
The test needs to exercise at least some of the program’s core features.

For example, you could write a minimal YAML with a single `command` named `hello`, run `ahoy` against it, and compare its standard output to `Hello world`. How about that?

Reply:
done